frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/ResetPasswordRequestHandler.java

```diff
@@ -108,40 +110,31 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
             }
             String persistentSessionId =
                     PersistentIdHelper.extractPersistentIdFromHeaders(input.getHeaders());
-            auditService.submitAuditEvent(
-                    FrontendAuditableEvent.PASSWORD_RESET_REQUESTED,
-                    userContext.getClientSessionId(),
-                    userContext.getSession().getSessionId(),
-                    userContext
-                            .getClient()
-                            .map(ClientRegistry::getClientID)
-                            .orElse(AuditService.UNKNOWN),
-                    AuditService.UNKNOWN,
-                    request.getEmail(),
-                    IpAddressHelper.extractIpAddress(input),
-                    AuditService.UNKNOWN,
-                    persistentSessionId);
 
```

Comment:
Only thing about moving the audit event is that you won't get an audit event if 'validatePasswordResetCount' generates an error, so audit wouldn't know if a user tries to reset their password 5 times and fails

Reply:
Resolved